ui/apps/platform/src/Containers/Vulnerabilities/VirtualMachineCves/aggregateUtils.ts

```diff
@@ -198,3 +201,44 @@ export function applyVirtualMachineCveTableFilters(
         return true; // passed all filter conditions
     });
 }
+
+export function applyVirtualMachineCveTableSort(
+    rows: CveTableRow[],
+    sortKey: string,
+    reversed: boolean
+): CveTableRow[] {
+    const comparator = (a: CveTableRow, b: CveTableRow) => {
+        let compareResult = 0;
+
+        switch (sortKey) {
+            case CVE_SORT_FIELD:
+                // Intl.Collator sorting would be better here, but it's not used in the backend
+                compareResult = a.cve.localeCompare(b.cve);
+                break;
+            case CVSS_SORT_FIELD:
+                compareResult = a.cvss - b.cvss;
+                break;
+            case CVE_EPSS_PROBABILITY_SORT_FIELD:
+                compareResult = a.epssProbability - b.epssProbability;
+                break;
+            case CVE_STATUS_SORT_FIELD:
+                compareResult = Number(a.isFixable) - Number(b.isFixable);
```

Comment:
Except for name or sometimes date that is meaningful to sort, or filter, or both, categorial data like fixability. is usually better to filter instead of to sort.

**CVE status** column looks like **not** sortable for Deployment, Image, or Node.

In case future backend sorting might not support it, what do you think to omit it from frontend sorting?